cyral/resource_cyral_access_rules.go

```diff
@@ -0,0 +1,292 @@
+	Name:       "RepositoryAccessRulesRead",
+	HttpMethod: http.MethodGet,
+	CreateURL: func(d *schema.ResourceData, c *client.Client) string {
+		return fmt.Sprintf("https://%s/v1/repos/%s/userAccounts/%s/accessRules",
+			c.ControlPlane,
+			d.Get("repository_id").(string),
+			d.Get("user_account_id").(string),
+		)
+	},
+	NewResponseData: func(_ *schema.ResourceData) ResponseData {
+		return &AccessRulesResponse{}
+	},
+}
+
+func resourceRepositoryAccessRules() *schema.Resource {
+	return &schema.Resource{
+		Description: "Manage access rules",
+		CreateContext: CreateResource(
+			ResourceOperationConfig{
+				Name:       "RepositoryAccessRulesCreate",
+				HttpMethod: http.MethodPost,
+				CreateURL: func(d *schema.ResourceData, c *client.Client) string {
+					d.SetId(
+						marshalComposedID(
+							[]string{
+								d.Get("repository_id").(string),
+								d.Get("user_account_id").(string),
+							},
+							"/",
+						),
+					)
+					return fmt.Sprintf("https://%s/v1/repos/%s/userAccounts/%s/accessRules",
+						c.ControlPlane,
+						d.Get("repository_id").(string),
+						d.Get("user_account_id").(string),
+					)
+				},
+				NewResourceData: func() ResourceData {
+					return &AccessRulesResource{}
+				},
+				NewResponseData: func(_ *schema.ResourceData) ResponseData {
+					return &AccessRulesResponse{}
+				},
+			},
+			ReadRepositoryAccessRulesConfig,
+		),
+		ReadContext: ReadResource(ReadRepositoryAccessRulesConfig),
+		UpdateContext: UpdateResource(
+			ResourceOperationConfig{
+				Name:       "RepositoryAccessRulesUpdate",
+				HttpMethod: http.MethodPut,
+				CreateURL: func(d *schema.ResourceData, c *client.Client) string {
+					return fmt.Sprintf("https://%s/v1/repos/%s/userAccounts/%s/accessRules",
+						c.ControlPlane,
+						d.Get("repository_id").(string),
+						d.Get("user_account_id").(string),
+					)
+				},
+				NewResourceData: func() ResourceData {
+					return &AccessRulesResource{}
+				},
+				NewResponseData: func(_ *schema.ResourceData) ResponseData {
+					return &AccessRulesResponse{}
+				},
+			},
+			ReadRepositoryAccessRulesConfig,
+		),
+		DeleteContext: DeleteResource(
+			ResourceOperationConfig{
+				Name:       "RepositoryAccessRulesDelete",
+				HttpMethod: http.MethodDelete,
+				CreateURL: func(d *schema.ResourceData, c *client.Client) string {
+
+					idPieces, err := unmarshalComposedID(d.Id(), "/", 2)
+					if err != nil {
+						panic("Failed to unmarshal access rules ID")
+					}
+					repoID := idPieces[0]
+					userAccountID := idPieces[1]
+
+					return fmt.Sprintf("https://%s/v1/repos/%s/userAccounts/%s/accessRules",
+						c.ControlPlane,
+						repoID,
+						userAccountID,
+					)
+				},
+			},
+		),
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Description: "ID of this resource in Cyral environment.",
+				Computed:    true,
+				Type:        schema.TypeString,
+			},
+
+			"repository_id": {
+				Description: "ID of the repository",
```

Comment:
```suggestion
				Description: "ID of the repository.",
```
Just a minor detail (the dot).

FYI, the description is copied into the markdown that goes to the Provider docs, so you can write markdown syntax here if you wish :).

Reply:
Yeah, thanks for catching this, I was pretty inconsistent with punctuation. Done.

And thanks for the suggestion about markdown!